uno/uno-js.md

```diff
@@ -213,13 +241,23 @@ description: "Check out uno-js, a Typescript library for array and string manipu
               An array with unique objects.
             </p>
           </div>
-          <div>
-            <iframe
-              src="https://codesandbox.io/embed/uno-jsremoveduplicated-6pmtz?fontsize=14&hidenavigation=1&theme=dark&previewwindow=console&view=split"
-              style="width:100%; height:300px; border:0; border-radius: 4px; overflow:hidden;"
-              title="uno-js/removeDuplicated" sandbox="allow-scripts allow-same-origin"></iframe>
-          </div>
         </div>
+```tsx
+{% raw %}
+import { removeDuplicated } from "uno-js";
+const x1 = { name: "Mike", age: 3, gender: "Male" };
+const x2 = { name: "Mike", age: 3, gender: "Male" };
+const y1 = { name: "John", age: 5, gender: "Male" };
+const y2 = { name: "John", age: 5, gender: "Male" };
+const array = [x1, x2, y1, y2];
+const unique = removeDuplicated([x1, x2, y1, y2], "name");
```

Comment:
It;'s best practice to add a comment next to the result with the output, instead of the random console.log